docs/application/dotnet/guides/app-management/event.md

```diff
@@ -0,0 +1,291 @@
+
+  - sound
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.SilentMode.EventName`           | `SystemEvents.SilentMode.Key`             | - `SystemEvents.SilentMode.ValueOn` <br> - `SystemEvents.SilentMode.ValueOff`                            | When  the ringtone has been changed to 0 or another mode. For example, if the call  slider has been changed to 0, `silent_mode` is `"on"`. Otherwise, `silent_mode` is `"off"`. | You  can get the current value with the `System.SystemSettings.SoundSilentModeEnabled` property. |
+
+  - vibration
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.VibrationState.EventName`       | `SystemEvents.VibrationState.StateKey`          | - `SystemEvents.VibrationState.StateValueOn` <br> - `SystemEvents.VibrationState.StateValueOff`                           | When the  vibration state has changed.   | You can get the current value using System.Information.SetCallback with the `http://tizen.org/runtimefeature/vibration` property. |
+
+  - screen
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.AutoRotateState.EventName` | `SystemEvents.AutoRotateState.StateKey` | - `SystemEvents.AutoRotateState.StateOn` <br> - `SystemEvents.AutoRotateState.StateOff`                      | When the screen autorotate state  has been  changed, for example, by the user toggling the display settings. | You can get the  current value with the `System.SystemSettings.DisplayScreenRotationAutoEnabled` property. |
+
+  - mobile
+
+    | Event name | Event data Key | Event data Value | Condition | Notes |
+    |-------|-------|--------|--------|--------|
+    | `SystemEvents.MobileDataState.EventName`     | `SystemEvents.MobileDataState.StateKey`       | - `SystemEvents.MobileDataState.StateValueOn` <br> - `SystemEvents.MobileDataState.StateValueOff`            | When the mobile data state has been changed,  for example, by the user toggling the network settings. | You can get the  current value with the `System.SystemSettings.Data3GNetworkEnabled` property. |
+    | `SystemEvents.DataRoamingState.EventName`    | `SystemEvents.DataRoamingState.StateKey`      | - `SystemEvents.DataRoamingState.StateValueOn` <br> - `SystemEvents.DataRoamingState.StateValueOff`                   | When the data roaming state  has been changed,  for example, by the user toggling the network settings. | You can get the current value using System.Information.SetCallback with the `http://tizen.org/runtimefeature/dataroaming` key. |
```

Comment:
How about `System.Information.SetCallback`? Applying font style to System.Information.SetCallback.